docs/HyperIndex/Advanced/performance/benchmarking.md

```diff
@@ -9,7 +9,7 @@ slug: /benchmarking
 
 ### Capturing the benchmark
 
-You can run `envio start --bench` to capture some benchmarking data while your indexer is running. This should not be run in production, since it holds each benchmark data point in memory and adds some overhead in writing to a file.
+You can run `pnpm envio start --bench` to capture some benchmarking data while your indexer is running. This should not be run in production, since it holds each benchmark data point in memory and adds some overhead in writing to a file.
```

Comment:
The only downside of dropping `envio` and having only `pnpm start` is that flags won't work in this case.

Reply:
I think pnpm is smart enough to pick up and add flags - from what I remember. If not I'm sure `pnpm start -- --bench` or similar would work. Should test it.

Reply:
Ah I see what you mean - we don't run the `envio start` command in the package.json. So here we should keep the word envio :thinking: Will leave this as `pnpm envio start`